CoreArrangementPlugins/release/modules/ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/uncollide-arrangement.md

```diff
@@ -0,0 +1,46 @@
+# Uncollide
+
+<table class="table table-striped">
+<thead>
+<tr class="header">
+<th>Constellation Action</th>
+<th>User Action</th>
+<th style="text-align: center;">Menu Icon</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>Run Uncollide</td>
+<td>Arrange -&gt; Uncollide</td>
+<td style="text-align: center;"><img src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/uncollide.png" alt="Uncollide Icon" /></td>
+</tr>
+
+<tr class="even">
+<td>Run Uncollide 3D</td>
+<td>Arrange -&gt; Uncollide 3D</td>
+<td style="text-align: center;"><img src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/uncollide3D.png" alt="Uncollide 3D Icon" /></td>
+</tr>
+</tbody>
+</table>
+
+
+Selected nodes are postioned such that they don't "collide" with one another. 
```

Comment:
might be good to explain what it means for nodes to be colliding (i.e. overlapping) and how it works (i.e. it expands the graph until the nodes no longer collide)